src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -457,6 +496,71 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
               </div>
             </div>
           </section>
+          {/**
+           * this part need to refactor
+           * this part should only display on cpo review and onward
+           * like final review, pending completion and complete
+           * right now we only have two status, cpo review and final review
+           */}
+          {pia.hasAddedPiToDataElements !== false &&
+          (pia.status === PiaStatuses.CPO_REVIEW ||
```

Comment:
please add any status checks to the state table to eliminate these checks all over the place